yb-voyager/cmd/assessMigrationCommandBulk.go

```diff
@@ -51,10 +59,18 @@ var assessMigrationBulkCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(assessMigrationBulkCmd)
 
+	const fleetConfigFileHelp = `
+Path to the CSV file with connection parameters for schema(s) to be assessed.
+Fields (case-insensitive): dbtype, hostname, port, service_name, sid, tns_alias, username, password, schema.
```

Comment:
let's keep the style consistent: `db_type`, `host_name` ?

Reply:
yeah sure.
I am just not sure about `hostname` and `username`, whether there should be any `-` in them or not.

Also just thinking whether we should following the same voyager flags here:
`hostname` --> `db_host`
`username` --> `db_user`